tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -194,6 +204,29 @@ Status TRTEngineOp::ConstructFunctionHandle(OpKernelContext* ctx) {
                           &native_func_);
 }
 
+Status TRTEngineOp::ConstructFunctionHandle(OpKernelConstruction* ctx) {
```

Comment:
Can we avoid duplication? 
e.g. we can share the code through a helper function that is called by both `ConstructFunctionHandle` and `ConstructFunctionHandle`.

Reply:
What is the difference between `ConstructFunctionHandle` and `ConstructFunctionHandle`?
I am wondering if we can get one from the other?

Reply:
One takes in an OpKernelContext and one takes an OpKernelConstruction. I couldn't find a relevant conversion. Should I do this through a templated helper function? Or should we pass in the things needed from the contexts in as parameters?